src/Testcontainers/Clients/TraceProgress.cs

```diff
@@ -15,7 +15,23 @@ public TraceProgress(ILogger logger)
 
     public void Report(JSONMessage value)
     {
-      this.logger.Progress(value.ProgressMessage);
+      this.LogProgressIfNotNullOrWhiteSpace(value.Stream);
```

Comment:
Can't we use a single logging command (pattern) for those three properties?

Reply:
We can, but I don't know if it makes sense. I added few screenshots from the state JSONMessage object bellow. Notice that every event generates exactly one message (only one property is filled). Because of that I'm not sure if it will be good to create such logs: "Stream: [null], ProgressMessage: [null], Status: Lorem Ipsum dadadada"